docker-compose-parts/hypervisor.yml

```
@@ -17,5 +17,6 @@ services:
       - /opt/isard/hypervisor/sshd_keys:/usr/local/etc/ssh:rw
       - /opt/isard:/isard:rw
       - /opt/isard-local/sockets/:/var/run/libvirt/
+      - /opt/isard/networks:/networks
```

What about use /opt/isard/hypervisor/networks instead of /opt/isard/networks?
```suggestion
      - /opt/isard/hypervisor/networks:/networks
```